circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  pre:
+    - cd /opt/circleci/.pyenv/plugins/python-build/../.. && git pull && cd -
```

Comment:
Do we need this? CI automatically checks out code

Reply:
CI asks for this :)

Reply:
It didn't before and in other projects :)  but anyway, if it works then no need to change.

Reply:
For some reason circleci wasn't able to find `3.5.5`